src/plugins/discover/public/__mocks__/saved_search.ts

```diff
@@ -17,6 +17,8 @@ export const savedSearchMock = {
   id: 'the-saved-search-id',
   title: 'A saved search',
   searchSource: createSearchSourceMock({ index: dataViewMock }),
+  columns: ['default_column'],
```

Comment:
ℹ️ Making the mocked saved search more realistic, as these are filled in automatically when discover loads the saved search